core/src/main/scala/org/http4s/Uri.scala

```diff
@@ -24,11 +24,11 @@ import org.http4s.util.option.ToOptionOps
   */
 // TODO fix Location header, add unit tests
 case class Uri(
-  scheme: Option[CaseInsensitiveString] = None,
-  authority: Option[Authority] = None,
-  path: Path = "",
-  query: Query = Query.empty,
-  fragment: Option[Fragment] = None)
+                scheme: Option[CaseInsensitiveString] = None,
```

Comment:
Slight preference for the old whitespace formatting. Just giving my 2 cents. It goes without saying that I wish we could move away from text to serialize code and stop debating whitespace.